### PR TITLE
GitMate CI Test

### PR DESCRIPTION
**Pull request by yukiisbored**

### CODE REVIEW

**Comment by yukiisbored** [comment]
Hello,

Welcome to Yuki's Playground.

Thanks :3

File: terrible.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import sys
+import os
+import time
+name='Yuki'
```
Comment (yukiisbored) [inline terrible.py:5]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -2,7 +2,9 @@
 import sys
 import os
 import time
-name='Yuki'
+name = 'Yuki'
+
+
 class Greeter:
     def greet(self):
         print('Hello, world!')
```

File: terrible.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class Greeter:
+    def greet(self):
+        print('Hello, world!')
```
Comment (yukiisbored) [inline terrible.py:8]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -6,9 +6,12 @@
 class Greeter:
     def greet(self):
         print('Hello, world!')
+
+
 class NameGreeter(Greeter):
-    def __init__(self,who='world'):
-        self.__who=who
+    def __init__(self, who='world'):
+        self.__who = who
+
     def greet(self):
         print('Hello, {}!'.format(self.__who))
 if __name__=='__main__':
```

File: terrible.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+class NameGreeter(Greeter):
+    def __init__(self,who='world'):
+        self.__who=who
+    def greet(self):
+        print('Hello, {}!'.format(self.__who))
+if __name__=='__main__':
+    Greeter().greet()
+    NameGreeter().greet()
+    NameGreeter(name).greet()
```
Comment (yukiisbored) [inline terrible.py:17]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -11,7 +11,9 @@
         self.__who=who
     def greet(self):
         print('Hello, {}!'.format(self.__who))
-if __name__=='__main__':
+
+
+if __name__ == '__main__':
     Greeter().greet()
     NameGreeter().greet()
     NameGreeter(name).greet()
```

File: terrible.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```
Comment (yukiisbored) [inline terrible.py:1]: This file contains unused source code.

*Origin: PyUnusedCodeBear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -1,7 +1,4 @@
 #!/usr/bin/env python3
-import sys
-import os
-import time
 name='Yuki'
 class Greeter:
     def greet(self):
```

**Comment by naveenkumarsangi** [comment]
unack https://github.com/yukiisbored/playground/pull/2/commits/bfec7416c94d8a1c4ed0dd74f708a77eb665833a
https://github.com/yukiisbored/playground/pull/2/commits/6a9ceead00ebc68ef52e0d083a334bdeeb0f4ae1

**Comment by naveenkumarsangi** [comment]
@gitmate-bot rebase

**Comment by yukiisbored** [comment]
@gitmate-bot rebase please

**Pull request closed without merging**